.github/worflows/publish-chart.yml

```diff
@@ -0,0 +1,32 @@
+# Kindly refer to https://github.com/helm/chart-releaser-action
+
+name: Publish Charts
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Configure Git
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+      - name: Install Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Run chart-releaser
+        uses: helm/chart-releaser-action@v1.2.0
+        env:
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
```suggestion
          CR_TOKEN: "${{ secrets.GH_TOKEN }}"
```

Reply:
For some reason I can't create a secret called "GITHUB_TOKEN" but I can create "GH_TOKEN"